frontend/packages/console-shared/src/components/badges/Badge.scss

```
@@ -5,7 +5,7 @@
   }
 }
 
-.tech-preview {
+.odc-preview-badge {
```

```suggestion
.osc-preview-badge {
```
Not part of the DevConsole package, so can't have 'odc' (aka Openshift Developer Console) prefix.

---

Changed.